tests/common/gcp_api/compute_test.py

```diff
@@ -117,6 +117,13 @@ def test_get_firewall_rules_errors(self, name, response, status,
         with self.assertRaises(expected_exception):
             list(self.gce_api_client.get_firewall_rules(self.project_id))
 
+
+    def test_get_global_operations(self):
+        """Test get_global_operations"""
+        http_mocks.mock_http_response(fake_compute.GLOBAL_OPERATIONS_RESPONSE)
+        results = self.gce_api_client.get_global_operations(
+            self.project_id, operation_id='operation-1234')
```

Comment:
Nit: Add a FAKE_OPERATION_ID variable to fake_compute_responses set to 'operation-1234' and then use fake_compute.FAKE_OPERATION_ID here.